### PR TITLE
fix bug

### PR DESCRIPTION
**Pull request by Theodore-Chatziioannou**
Fix for #263 .

## Checklist

Any checks which are not relevant to the PR can be pre-checked by the PR creator.
All others should be checked by the reviewer(s).
You can add extra checklist items here if required by the PR.

- [x] CHANGELOG updated
- [x] Tests added to cover contribution
- [x] Documentation updated

### CODE REVIEW

File: tests/test_03_read_matsim.py [hunk 1]
```diff
@@ -210,3 +210,14 @@ def test_get_attributes_from_person():
     assert pid == "chris"
     assert attributes["hid"] == "A"
     assert attributes["vehicles"] == {"car": "chris"}
+
+
+def test_get_float_attribute_from_person():
```
Comment (mfitz) [inline tests/test_03_read_matsim.py:215]: When you ran this test before fixing the bug, did it throw an exception?

**Review by mfitz — APPROVED** [review]

File: tests/test_03_read_matsim.py (hunk 1 above)
Comment (Theodore-Chatziioannou) [reply tests/test_03_read_matsim.py:215]: yep

Comment (Theodore-Chatziioannou) [reply tests/test_03_read_matsim.py:215]: it was tring to cast a lxml element as float (`float(attr)`), instead of the element value (`attr.text`)

**Review by brynpickering — APPROVED** [review]
Good catch!

**Comment by brynpickering** [comment]
@Theodore-Chatziioannou you need to run pre-commit locally!

**Comment by Theodore-Chatziioannou** [comment]
> @Theodore-Chatziioannou you need to run pre-commit locally!

`pre-commit run` ? It is not changing anything..

**Comment by brynpickering** [comment]
hmm, pre-commit should have fixed this issue when you first committed (assuming you have run `pre-commit install` within your pam environment). Anyway, `pre-commit run -a` should force it to run it on the files you've already committed.

**Pull request merged**